scripts/blt/drush-config.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+REPOROOT="$1"
+
+DRUSH8="\"$REPOROOT/vendor-bin/drush-8/\""
+
+DRUSH9="\"$REPOROOT/vendor-bin/drush-9/vendor/bin\""
+
+if [ "`basename "/$SHELL"`" = "zsh" ]; then
+  DETECTED_PROFILE="$HOME/.zshrc"
+elif [ -f "$HOME/.bash_profile" ]; then
+  DETECTED_PROFILE="$HOME/.bash_profile"
+elif [ -f "$HOME/.bashrc" ]; then
+  DETECTED_PROFILE="$HOME/.bashrc"
+elif [ -f "$HOME/.profile" ]; then
+  DETECTED_PROFILE="$HOME/.profile"
+  fi
+
+if [ ! -z "$DETECTED_PROFILE" ]; then
+  if [ "`grep 'BLT GENERATED DRUSH SCRIPT PATH ALIASES' $DETECTED_PROFILE`" ]; then
+    echo "You have existing drush executable overrides installed."
+    echo "Remove modifications and run this task agin to generate new $DETECTED_PROFILE config."
+    exit
+    fi
+
+    echo "Writing config to $DETECTED_PROFILE"
+
+    echo "#BLT GENERATED DRUSH SCRIPT PATH ALIASES" >> $DETECTED_PROFILE
```

Comment:
Let's just drop all of this into a separate file that we can tack onto $DETECTED_PROFILE.